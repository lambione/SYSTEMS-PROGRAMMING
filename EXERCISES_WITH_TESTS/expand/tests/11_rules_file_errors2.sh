#!/bin/bash
"$1" tests/rules1.txt tests/BAD_RULES <<EOF
the program must terminate without printing this line.
EOF
# we are expecting the program to fail, so...
test "$!" != 0


    
