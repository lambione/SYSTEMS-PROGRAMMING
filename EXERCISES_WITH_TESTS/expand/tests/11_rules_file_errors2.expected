invalid rules file tests/BAD_RULES
